brands/shop/variety_store.json

```diff
@@ -58,6 +58,17 @@
       "shop": "variety_store"
     }
   },
```

Comment:
Bright house really shouldn’t be a variety store since it’s a rent to own shop. OSM unfortunately doesn’t have an appropriate tag for that which is probably why it’s tagged differently throughout the UK. In the US we’ve used electronics and furniture tags for similar stores but those are also incorrect.

Reply:
I did go back and forth with how to tag BrightHouse, but came to wonder would the method of payment have a bearing on the OSM tagging, as opposed to the goods being sold by the shop?

Isn't _Rent to Own_ simply another way of paying on credit / account, similar to how I could go into the John Lewis department store, buy a new bed, and pay it off in 12 month instalments, either via a credit card transaction, or a credit agreement with John Lewis??

Reply:
I'd probably just call it a `shop=department_store` if they sell furniture, electronics, appliances.  `variety_store` is mostly used for dollar stores like Poundland/Poundworld, etc.

Reply:
As recommended, I have moved BrightHouse into the department_store.json file, and removed it from variety_store.json, and I think GitHub has automatically pulled that update in to this request?